client/src/components/Viewer/Viewer.js

```diff
@@ -55,6 +55,9 @@ function Viewer(props) {
         setIsConnected(true);
         socket.emit('addUser', {
           token: JSON.parse(localStorage.getItem('dfs-user'))?.['token'],
+          user_id: JSON.parse(localStorage.getItem('dfs-user'))?.['user'][
+            'user_email'
+          ],
```

Comment:
Consider renaming the `user_id` field to `user_email` for clarity, as it directly represents the user's email rather than an ID.


    *This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*